admin_guide/managing_networking.adoc

```diff
@@ -670,24 +666,55 @@ redirecting rules.
 proxy. Note that this is specified as an environment variable in the
 HTTP proxy container, not with the other environment variables in the
 init container.
-====
++
+You can specify any of the following for the `EGRESS_HTTP_PROXY_DESTINATION`
+value. You can also use `*`, meaning "allow connections to all remote
+destinations". Each line in the configuration specifies one group of connections
+to allow or deny:
++
+- An IP address (eg, `192.168.1.1`) allows connections to that IP address.
+- A CIDR range (eg, `192.168.1.0/24`) allows connections to that CIDR range.
+- A host name (eg, `www.example.com`) allows proxying to that host.
+- A domain name preceded by `\*.` (eg, `*.example.com`) allows proxying to that domain and all of its subdomains.
+- A `!` followed by any of the above denies connections rather than allowing them
+- If the last line is `*`, then anything that hasn't been denied will be allowed. Otherwise, anything that hasn't been allowed will be denied.
 
-You can specify a variety of things for
-`EGRESS_HTTP_PROXY_DESTINATION`. The simplest is to just specify `*`,
-meaning "allow connections to all remote destinations". Other than
-that, each line specifies one group of connections to allow or deny:
+. Ensure other pods can find the pod's IP address by creating a service to point
+to the egress router:
++
+[source, yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: egress-1
+spec:
+  ports:
+  - name: http
+    port: 8080 <1>
+  - name: https
+    port: 443
+  type: ClusterIP
+  selector:
+    name: egress-1
+----
+<1> Ensure the `http` port is always set to `8080`.
```

Comment:
@danwinship @weliang1 So, delete the whole https section, or just replace the 443 with 8080? If I'm adding the `https-proxy` variable below, does that mean it's required here as well?